Lib/idlelib/outwin.py

```diff
@@ -1,43 +1,115 @@
+"""Editor window that can serve as an output file.
+"""
+
 import re
 
-from tkinter import *
-import tkinter.messagebox as tkMessageBox
+from tkinter import messagebox
 
 from idlelib.editor import EditorWindow
 from idlelib import iomenu
 
 
-class OutputWindow(EditorWindow):
+file_line_pats = [
```

Comment:
I moved the _file_line_helper and these values to the module level per Terry's suggestion on the bug tracker.